.editorconfig

```diff
@@ -0,0 +1,21 @@
+; EditorConfig to support per-solution formatting.
+; Use the EditorConfig VS add-in to make this work.
+; http://editorconfig.org/
+
+; This is the default for the codeline.
+root = true
+
+[*]
+end_of_line = CRLF
+
+[*.{cs,props,targets,nuspec}]
+indent_style = space
+indent_size = 4
+
+[*.{config}]
+indent_style = space
+indent_size = 2
+
+[*.{csproj,xml,resx}]
```

Comment:
All xml-based formats should be in the same section, namely: xml, props, targets, csproj, nuspec